source/portable/NetworkInterface/Common/phyHandling.c

```diff
@@ -626,12 +627,12 @@ BaseType_t xPhyStartAutoNegotiation( EthernetPhy_t * pxPhyObject,
 
             if( ( ulRegValue & phyBMSR_LINK_STATUS ) != 0 )
```

Comment:
If you don't mind:
~~~diff
-    if( ( ulRegValue & phyBMSR_LINK_STATUS ) != 0 )
+    if( ( ulRegValue & phyBMSR_LINK_STATUS ) != 0U )
~~~